drivers/clk/renesas/rzg2l-cpg.c

```diff
@@ -550,27 +596,39 @@ rzg2l_cpg_sd_mux_clk_register(const struct cpg_core_clk *core,
 
 static unsigned long
 rzg2l_cpg_get_foutpostdiv_rate(struct rzg2l_pll5_param *params,
-			       unsigned long rate)
+			       unsigned long rate,
+				   struct rzg2l_cpg_priv *priv)
```

Comment:
Remove extra spaces between struct parameters for better alignment and readability